src/v3/test/integration/__snapshots__/webauthn-enroll.test.tsx.snap

```diff
@@ -95,10 +96,10 @@ exports[`webauthn-enroll should render form 1`] = `
                   class="ods-4o5zYQ ods-76eppy ods-6lar65 ods-1OsrNt ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs"
                 >
                   <span
-                    aria-label="Waiting for input"
```

Comment:
intentional change?

Reply:
This was generated by this key `renderers.waiting.indicator.label` which was not a real v2 i18n key. So it is removed now that we are using loc. We will have to work with translator, design and product to generate net new keys.